hw03-annotaions/src/main/java/ru/otus/TestDemo.java

```diff
@@ -0,0 +1,10 @@
+package main.java.ru.otus;
```

Comment:
`main.java` не стоит включать в имя пакета.

```suggestion
package ru.otus;
```

Папки main и java не входят в иерархию пакетов.
Пакеты начинаются с того, что уже внутри main\java 

И импорты везде поправьте